src/data/userData.tsx

```diff
@@ -134,7 +134,18 @@ export const Users: User[] = [
       "https://github.com/dhairyagothi/50_days_50_web_project/tree/Main/public/TO_DO_LIST",
     tags: ["html", "css", "javascript"],
   },
-
+  
+  // Portfolio
+  {
+    title: "Portfolio",
+    description:
```

Comment:
@kartikmehta18 The description for your Portfolio project is too long. In order to maintain a uniform layout for the cards on the main page, please limit your project description to 120 characters or less.

Reply:
but the given  description in code is only 80 words I count on google

Reply:
> but the given description in code is only 80 words I count on google

@kartikmehta18, Please rewrite the description for your Portfolio project to be 120 characters or less. My meaning is characters, not words.

Reply:
plz check now it is 80 words
Portfolio can showcase your skills ,certificateand serve as part of your resume.